### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by emmatrain**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -99,22 +101,233 @@ header a:hover {
...
+
+@media only screen and (max-width: 600px) {
+	
+	.projects-container article {
+		width: 100%;
+	}
+	
+	.show {
+		display: none;
+	}
```
Comment (MattW-R) [inline styles.css:243]: At 650px your widescreen-navbar elements start falling of the page to the left consider applying this and .responsive-nav (line 252-258) at 650px instead?

Have made a few more responsiveness-related comments below - May want to check they all work nicely together at different widths once implemented.

Also could consider more semantic class name?

File: index.html [hunk 2]
```diff
@@ -5,23 +5,31 @@
```
Comment (MattW-R) [inline index.html:5]: It wont let me comment above ^^ - You use double quotes most the time, may want to use lang="en" for consistency

File: index.html [hunk 3]
```diff
@@ -30,36 +38,152 @@
...
+<!-- FOOTER SECTION -->
+
+		<section id="contact">
+	
+			<footer>
+				<li><a href="index.html">emtrain7@gmail.com</a></li>
+				<li><a href="#about">GitHub</a></li>
+				<li><a href="#projects">Twitter</a></li>
+				<li><a href="#contact">Blog</a></li>
+				<li><a href="#contact">LinkedIn</a></li>
```
Comment (MattW-R) [inline index.html:183]: May want to replace placeholder links? (Unless you are not ready for that yet.)

File: styles.css [hunk 4]
```diff
@@ -99,22 +101,233 @@ header a:hover {
...
+	font-family: 'Montserrat', sans-serif;
+	color: #ed4264; /*dark pink*/
+	font-weight: 500;
+	font-size: 2rem;
+}
+
+#about h3, 
+#projects h3 {
+	font-family: 'Montserrat', sans-serif;
+	color: #0000;
```
Comment (MattW-R) [inline styles.css:138]: `#0000` = `#00000000` = transparent - You probably wanted `#000` for black as the text is currently not visible

File: styles.css [hunk 5]
```diff
@@ -33,26 +42,19 @@ body {
...
 	text-decoration: none;
 }
 
-nav a:link    {color:#ed4264;} /*dark pink*/
-nav a:visited {color:#ed4264;} /*dark pink*/
-nav a:hover   {color:#FFFFFF;}
-nav a:active  {color:#ed4264;} /*dark pink*/
+nav a, footer a:link    {color:#ed4264;} /*dark pink*/
+nav a, footer a:visited {color:#ed4264;} /*dark pink*/
+nav a, footer a:hover   {color:#FFFFFF;}
```
Comment (MattW-R) [inline styles.css:55]: May want to pick different hover colour in footer anchors as it is hard to read when hovering

File: index.html [hunk 6]
```diff
@@ -30,36 +38,152 @@
...
+			<section class=responsive-intro>
+				<h3>How do you do? I'm Emma.</h3>
+				<h1>I make <span>{ things }</span> on the internet.</h1>
+				<h2>I'm a full-stack web developer student at IO Academy.</h2>
+				<h2>I say things on my <a href="https://dev.to/emma_train" target="_blank">blog</a>.</h2>
+			</section>
+
+		</div>	  
+	</div>	
+	</div>
```
[the remark this answers is not in the document]
Comment (emmatrain) [reply index.html:84]: I deleted these and then they removed the responsive navigation. who knows haha.

[the remark this answers is not in the document]
Comment (emmatrain) [reply index.html:84]: When I delete they much up the responsive nav, i think they close the diagnonal box and header wrapper

File: styles.css [hunk 7]
```diff
@@ -99,22 +101,233 @@ header a:hover {
...
+	padding-right: 20px;
+}
+
+
+/*MAIN STYLING*/
+
+#about h1, 
+#projects h1 {
+	font-family: 'Montserrat', sans-serif;
+	color: #ed4264; /*dark pink*/
```
[the remark this answers is not in the document]
Comment (emmatrain) [reply styles.css:123]: I will have a think and a play with colours after lunch. If i do it now I'll hold everything up haha.

File: styles.css [hunk 8]
```diff
@@ -99,22 +98,270 @@ header a:hover {
...
+		padding: 8px;
+	}
+	
+	.responsive-intro {
+		padding: 30px 0 0 0;
+		margin: 10px 100px 100px;
+		display: block;
+		font-family: 'Montserrat', serif;
+		
+	}
```
Comment (MattW-R) [inline styles.css:270]: Apply this in max-width: 1160px instead

File: styles.css [hunk 9]
```diff
@@ -99,22 +98,270 @@ header a:hover {
...
+	font-size: 1.5rem;
+}
+
+/*RESPONSIVE STYLYING*/
+
+@media only screen and (max-width: 600px) {
+	
+	.projects-container article {
+		width: 100%;
+	}
```
Comment (MattW-R) [inline styles.css:242]: Apply this in max-width: ~1190px instead

File: styles.css [hunk 10]
```diff
@@ -99,22 +98,270 @@ header a:hover {
...
+	.responsive-nav a {
+		color: white;
+		font-size: 2rem;
+		width: 650px;
+	}
+	
+	.responsive-nav li {
+		list-style-type: none;
+		padding: 8px;
+	}
```
Comment (MattW-R) [inline styles.css:262]: Apply this in max-width: 650px instead

**Pull request merged**